5 Errors during the build with the latest celebrate updates

After pulling the latest changes for celebrate a service cannot build - the errors are:

```
> ttsc -p .

node_modules/celebrate/lib/index.d.ts:26:77 - error TS2344: Type 'P' does not satisfy the constraint 'Params'.
  Type 'P' is not assignable to type 'ParamsArray'.

26   (joiOpts: ValidationOptions, requestRules: SchemaOptions): RequestHandler<P, ResBody, ReqBody, ReqQuery>;
                                                                               ~

node_modules/celebrate/lib/index.d.ts:40:23 - error TS2344: Type 'P' does not satisfy the constraint 'Params'.
  Type 'P' is not assignable to type 'ParamsArray'.

40     ): RequestHandler<P, ResBody, ReqBody, ReqQuery>;
                         ~

node_modules/celebrate/lib/index.d.ts:84:196 - error TS2344: Type 'P' does not satisfy the constraint 'Params'.
  Type 'P' is not assignable to type 'ParamsArray'.

84 export declare function celebrate<P=ParamsDictionary, ResBody=any, ReqBody=any, ReqQuery=Query>(requestRules: SchemaOptions, joiOpts?: ValidationOptions, opts?: CelebrateOptions): RequestHandler<P, ResBody, ReqBody, ReqQuery>;
                                                                                                                                                                                                      ~

node_modules/celebrate/lib/index.d.ts:94:146 - error TS2344: Type 'P' does not satisfy the constraint 'Params'.
  Type 'P' is not assignable to type 'ParamsArray'.

94 export declare function errors<P=ParamsDictionary, ResBody=any, ReqBody=any, ReqQuery=Query>(opts?: { statusCode: number }): ErrorRequestHandler<P, ResBody, ReqBody, ReqQuery>;
                                                                                                                                                    ~

src/http/nodegen/utils/createHttpExceptionFromErr.ts:15:61 - error TS2550: Property 'fromEntries' does not exist on type 'ObjectConstructor'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2019' or later.

15     httpException = new UnprocessableEntityException(Object.fromEntries(error.details));
                                                               ~~~~~~~~~~~


Found 5 errors.

```

--- a/package.json
+++ b/package.json
@@ -51,7 +51,7 @@
   "devDependencies": {
     "@types/command-line-args": "^5.0.1",
     "@types/cors": "^2.8.10",
-    "@types/express": "4.17.7",
+    "@types/express": "^4.17.13",
     "@types/express-form-data": "^2.0.2",
     "@types/fs-extra": "^9.0.12",
     "@types/jest": "^26.0.24",

